scripts/mkcloud

```diff
@@ -1291,6 +1305,12 @@ function sanity_checks
         fi
     fi
 
+    if [[ $wantedcmds =~ "setupironicnodes" ]]; then
```

Comment:
My point is, that mkcloud can be called step by step:
```
mkcloud step1
mkcloud step2
mkcloud setupironicnodes
mkcloud step7
....
```
With this `$wantedcmds` only holds the current step. So the check whether there is some step in the `$wantedcms` to be done later will not catch anything. It will only catch it if the respective step is run - but then its already too late (wasted time).

So please check for something other than the wantedcms.